src/lexer.coffee

```diff
@@ -337,6 +382,8 @@ exports.Lexer = class Lexer
           offset: match.index + match[1].length
       when match = @matchWithInterpolations HEREGEX, '///'
         {tokens, index} = match
+        comments = @chunk[0...index].match /\s+(#(?!{).*)/g
+        @commentToken comment for comment in comments if comments
```

Comment:
I'm not sure I understand the loop here. Probably missing something obvious, but since `comments` is the result of `RegExp#match`, will this not add `comment` twice, once with leading whitespace and once without?

```
coffee> console.log i for i in ' a'.match(/ (a)/)
 a
a
```